### DISCUSSION TITLE
Error al validar cobertura del proyecto (Macowins)

### DISCUSSION THREAD

**Discussion by GonzaloMartinezSa** [dds-jv/foro#345] [Consultas generales] [answered]
Lo hice en IntelliJ y corrí los test con maven y todo bien. El tema es que al correr el verify, en lo de cobertura me sale este error: 
```
[INFO] --- jacoco-maven-plugin:0.8.2:check (Validar Cobertura) @ ejercicio ---
[INFO] Loading execution data file C:\Users\gonza\2022-tpi-macowins-GonzaloMartinezSa\target\jacoco.exec
[INFO] Analyzed bundle 'ejercicio' with 10 classes
[WARNING] Rule violated for package ar.edu.utn.frba.dds.macowins: lines covered ratio is 0.36, but expected minimum is 0.60
for details. -> [Help 1]
[ERROR]
```
Seguí eso que aparece en el ReadMe (hablo del que aparece en el repo del tpi de macowins) para configurar el IDE sin problemas, así que no sé qué es esto. Tampoco sé cuán importante es que se cumpla, o sea, si ya lo puedo entregar o no.

Saludos.

**Comment by gastonprieto** [comment]
Buenas @GonzaloMartinezSa 

si es importante que se cumpla. fijate esta linea:

```
[WARNING] Rule violated for package ar.edu.utn.frba.dds.macowins: lines covered ratio is 0.36, but expected minimum is 0.60
```

Lo que te dice ahí es que te se espera que el porcentaje de cobertura (test que cubren lineas de codigo) sea superior al 60%, así que faltarían agregar casos de prueba para esta solución

**Comment by GonzaloMartinezSa** [comment, marked as answer]
Ahhhhhhh o sea que lo que faltaría es agregar más casos de prueba en la parte de tests? Si es eso, ahora lo hago, gracias.

**Reply by gastonprieto** [reply]
exactamente, pero casos de prueba que prueben lugares que no hayan probado los otros test.
